include/neml2/tensors/WR2.h

```diff
@@ -61,4 +63,14 @@ class WR2 : public VecBase<WR2>
   /// Derivative of the exponential map
   R2 dexp() const;
 };
+
+/// Shortcut product a_ik b_kj - b_ik a_kj with both SR2
+WR2 product_abmba(const SR2 & a, const SR2 & b);
```

Comment:
Similar questions as in `SR2.h`:
1. Should we just pass in `b` instead of both `a` and `b`?
2. Can we rename this to something like `skewify`?

Reply:
This wasn't a static method, but a separate binary operator.  I'll move it to math and think of a better name.